packages/framework/esm-styleguide/src/page-header/page-header.component.tsx

```diff
@@ -0,0 +1,37 @@
+/** @module @category UI */
+import React from 'react';
+import styles from './page-header.module.scss';
+import { useTranslation } from 'react-i18next';
+import { useSession } from '@openmrs/esm-framework';
+
+interface PageHeaderProps {
+  dashboardTitle: string;
+  illustration: React.ReactElement;
+  utilities?: React.ReactNode;
+  className?: string;
+}
+
+export const PageHeader: React.FC<PageHeaderProps> = ({ dashboardTitle, illustration, utilities, className }) => {
+  const { t } = useTranslation();
+  const session = useSession();
+  const location = session?.sessionLocation?.display;
+
+  const combineWords = (sentence: string): string => {
+    const lowerCaseWords = (match: string) => match.toLowerCase();
+    const capitalizeLetter = (_, group1: string) => group1.toUpperCase();
+    return sentence.replace(/\b\w+/g, lowerCaseWords).replace(/\s(\w)/g, capitalizeLetter);
+  };
+
+  return (
+    <div className={className ? className : styles.pageHeader}>
```

Comment:
I would be very surprised if passing `className` to a React component caused it not to apply its existing styles. Let's have `className` be additional to the default style. This uses the [`classNames`](https://www.npmjs.com/package/classnames) function.

```suggestion
    <div className={classNames(styles.pageHeader, className)}>
```